rpc/build.rs

```diff
@@ -32,4 +32,9 @@ fn main() {
             &["api/protobuf/v1"],
         )
         .unwrap_or_else(|e| panic!("Registration protobuf compilation failed: {}", e));
+
+    tonic_build::configure()
+        .build_server(true)
+        .compile(&["api/protobuf/ha_cluster_agent.proto"], &["api/protobuf"])
```

Comment:
Should this follow the new protobuf versioning? in v1?
Also, is this a control-plane only api? If so the proto file should live instead in control-plane/grpc/proto/v1,

Reply:
It will be control-plane only. I will move it in control-plane.